### ISSUE TITLE
Kodi notification triggers library cleaning when video is playing (even if always update is off)

### ISSUE THREAD

**Issue by maitikeisi** [Sonarr/Sonarr#5387]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When new media gets added via sonarr during video playback and Kodi gets notified of it, it triggers kodi to do a clean library even when "always update" isn't selected. (library update doesn't get triggered)

### Expected Behavior

I would expect that library cleaning is also skipped when video is playing as it causes unwanted distraction. Only library update is skipped at the moment.

### Steps To Reproduce

My kodi notification settings are as follows:
![2023-01-21 20_28_22-Connect Settings - Sonarr — Mozilla Firefox](https://user-images.githubusercontent.com/1008507/213881717-6ebb9778-2072-4bec-a54c-4464aed5509c.png)
![2023-01-21 20_27_53-Connect Settings - Sonarr — Mozilla Firefox](https://user-images.githubusercontent.com/1008507/213881685-93d38d28-dde2-4fc5-8162-1166a32f1c6e.png)

### Environment

```markdown
- OS: Ubuntu 22.04.1 LTS
- Sonarr: 4.0.0.358
- Docker Install: No
- Using Reverse Proxy: No
- Browser:
```

### What branch are you running?

Develop

### Trace Logs?

N/A

### Anything else?

I did find similar issue that was posted in 2018 and it was fixed back then (maybe?), but looks like it has regressed since then.
https://github.com/Sonarr/Sonarr/issues/2773

I did take a look at the current XbmcService.cs file, and to me it looks like library clean doesn't trigger the function to check if video is playing or not.

**Comment by Qstick** [comment]
Kodi version?

**Comment by maitikeisi** [comment]
> Kodi version?
Kodi 20.0 (Nexus)
